src/main/java/bio/terra/service/kubernetes/KubeService.java

```diff
@@ -120,6 +120,16 @@ public boolean stopPodListener(TimeUnit timeUnit, long joinWait) {
         return true;
     }
 
+    public int getActivePodCount() {
+        if (podListener != null) {
+            int podCount = podListener.getActivePodCount();
+            return podCount;
+        }
+        int defaultPodCount = 1;
```

Comment:
nit: I don't think that the default value can ever be anything except one, so this is maybe more care than it deserves...

Reply:
@ddietterich I added the log so that we could get a sense as to whether the pods were actually getting counted or if the default value was provided. But, I'm not sure how useful this will be in practice. Thoughts?

Reply:
Leave it be. It was a nit. We can remove it later if it gets boring to see the logging :)